command/agent/config/config.go

```diff
@@ -155,12 +155,6 @@ func LoadConfig(path string) (*Config, error) {
 		}
 	}
 
-	if result.AutoAuth != nil {
```

Comment:
Sorry, I was just re-checking the changes and I wonder if we want to keep a check like this?  If they have neither sinks, nor cache, nor templates, that seems like an invalid configuration that should return an error.

Reply:
That's a good question - I'm trying to think of a use case where you'd auto-auth and keep a token alive, but do nothing with it at all, and I can't think of one. I think if you are doing auto-auth and have no sinks, have no cache using an auto auth token and have no templates to render, yeah, that should be an error. I'll update this branch for that.